File: src/main/scala/FizzBuzz.scala
object FizzBuzz {

  def fizzbuzz(e: Int ): String =  if (e % 15 == 0) "FIZZBUZZ"
    else if (e % 3 == 0) "FIZZ"
    else if (e % 5 == 0) "BUZZ"
    else s"$e"

  def main(args: Array[String]) = {
    1 to 100 map(fizzbuzz) foreach(e => print(s"$e ")) 
    println
  }
}

File: src/main/kotlin/FizzBuzz.kt
fun fizzbuzz(i: Int):String {
	if (i % 15 == 0)
		return "FizzBuzz"
	else if (i % 3 == 0)
		return "Fizz"
	else if (i % 5 == 0)
		return "Buzz"
	else
		return i.toString()
}

fun main (args: Array<String>) {
        (1..100).map {fizzbuzz(it)}.forEach {print("$it ")}
	println()
}

